M2/cmake/check-libraries.cmake

```diff
@@ -80,9 +80,12 @@ if(WITH_TBB)
 endif()
 
 if(WITH_FFI)
-  find_package(FFI REQUIRED QUIET)
-  execute_process(COMMAND pkg-config --modversion libffi
-    OUTPUT_VARIABLE LIBFFI_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
+	# find_package(FFI REQUIRED QUIET)
+	# execute_process(COMMAND pkg-config --modversion libffi
+	# OUTPUT_VARIABLE LIBFFI_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
It's okay to leave these commented, but use two spaces for indenting instead of tabs.

Reply:
OK, done